src/app/shared/helper-services/update-data.service.ts

```diff
@@ -91,6 +91,14 @@ export class UpdateDataService {
         return fsat;
     }
 
+    addFsatOperations(fsat: FSAT): FSAT {
```

Comment:
You'll need to call this in the `updateFsat()` function. do a check on the baseline and the modification for the fsatOperations object